test/resources/template-generator-vdm/.npmrc

```diff
@@ -0,0 +1 @@
+@sap:registry=https://npm.sap.com
```

Comment:
I dont think this is necessary if the top .npmrc is corrected.

Reply:
yes, removed.